ciphers/rail_fence_cipher.cpp

```diff
@@ -0,0 +1,274 @@
+/*
```

Comment:
```suggestion
/**
```

doc-comment blocks must begin with `/**` and then with `*/`. Please update other locations as well.